components/HomeButton.js

```diff
@@ -35,10 +34,10 @@ const styles = StyleSheet.create({
         textAlign: 'center',
         fontSize: 30,
         minWidth: 165,
-        minHeight: 220,
+        minHeight: 20,
         padding: 10,
         left: 5,
-        top: 85,
+        top: 55,
```

Comment:
have u checked to make sure that all these changes do not affect other screen sizes? changing existing values by a lot seems alarming

Reply:
yes, looks good on bigger and smaller screens.